libraries/lib-playable-track/PlayableTrack.cpp

```diff
@@ -189,9 +189,9 @@ EnumSetting<SoloBehavior> TracksBehaviorsSolo{
    wxT("/GUI/Solo"),
    {
       ByColumns,
-      { XO("Multi-track"), XO("Simple"),  XO("None") },
-      { wxT("Multi"),     wxT("Simple"), wxT("None") }
+      { XO("Multi-track"), XO("Simple")},
+      { wxT("Multi"),     wxT("Simple")}
    },
-   0, // "Simple"
-   { SoloBehaviorMulti, SoloBehaviorSimple, SoloBehaviorNone },
+   0, // "Multi-track"
+   { SoloBehaviorMulti, SoloBehaviorSimple },
```

Comment:
Test the case that you select "None" in a previous version of Audacity, then open this one, and confirm that the preference defaults back to Multi-track.

Step `EnumSettingBase::Migrate` in the debugger to see it happen.